qrcode/main.py

```diff
@@ -103,7 +97,7 @@ def __init__(
     def version(self) -> int:
         if self._version is None:
             self.best_fit()
-        return cast(int, self._version)
+        return cast("int", self._version)
```

Comment:
Why is this change necessary? AFAIK `cast(SomeConcreteType, ...)` is fine?

Reply:
It's for consistency, I dont mind it. 

https://docs.astral.sh/ruff/rules/runtime-cast-value/